doh-server/config.go

```diff
@@ -42,6 +42,8 @@ type config struct {
 	Verbose          bool     `toml:"verbose"`
 	DebugHTTPHeaders []string `toml:"debug_http_headers"`
 	LogGuessedIP     bool     `toml:"log_guessed_client_ip"`
+	LocalIPFilter    bool     `toml:"ecs_allow_non_global_ip"`
+	ECSFullSubnet    bool     `toml:"ecs_use_precise_ip"`
```

Comment:
You haven't changed these variable names.
It is my fault to let these code be merged into the codebase.
Please also fix the variable names `LocalIPFilter` and `ECSFullSubnet`,
and match the behaviors of them.

This is to fix issue #97